config/config.go

```diff
@@ -167,15 +175,21 @@ type Influx struct {
 	Password string `mapstructure:"password"`
 }
 
+func (cfg *Influx) validateAndLog() {
+	log.Infof("config.metrics.influx.host: %s", cfg.Host)
+	log.Infof("config.metrics.influx.database: %s", cfg.Database)
+	// This intentionally skips username and password for security reasons.
+}
+
 type PrometheusMetrics struct {
 	Port             int    `mapstructure:"port"`
 	Namespace        string `mapstructure:"namespace"`
 	Subsystem        string `mapstructure:"subsystem"`
 	TimeoutMillisRaw int    `mapstructure:"timeout_ms"`
 }
 
-func (cfg *Influx) validateAndLog() {
-	log.Infof("config.metrics.influx.host: %s", cfg.Host)
-	log.Infof("config.metrics.influx.database: %s", cfg.Database)
+func (cfg *PrometheusMetrics) validateAndLog() {
+	log.Infof("config.metrics.prometheus.namespace: %s", cfg.Namespace)
+	log.Infof("config.metrics.prometheus.subsystem: %s", cfg.Subsystem)
 	// This intentionally skips username and password for security reasons.
```

Comment:
Does prometheus even have a user name and password option?

Reply:
I'll correct this soon

Reply:
Corrected